nailgun/entities.py

```diff
@@ -1042,6 +1042,7 @@ def __init__(self, server_config=None, **kwargs):
             'attach_stdin': entity_fields.BooleanField(),
             'attach_stdout': entity_fields.BooleanField(),
             'command': entity_fields.StringField(
+                default='top',
```

Comment:
Using top is ok. Thanks for giving more context.

Let's merge and if needed in the future we update to a list of some commands to NailGun choose.